examples/dynamo/eg6/alg.x90

```diff
@@ -1,3 +1,38 @@
+! -----------------------------------------------------------------------------
```

Comment:
Please could you add the MO copyright here too (and change the Copyright... line below to say "Modifications copyright..."). Thanks!

Reply:
Done.